### ISSUE TITLE
Unable to open protected notes after aborting password screen using Android's back button

### ISSUE THREAD

**Issue by PartialVolume** [standardnotes/mobile#299]
This bug is 100% reproducible. Standard Notes V3.0.22. HTC Pro 20. Android V10. Build no 1.01.2401.1

How to reproduce:
You will need an already protected note, if you haven't got any protected notes then write a note and then protect it. Exit the note back to your list of notes. Next, click on any protected note. You are presented with a screen that asks you to enter your password. Now instead of entering your password you abort this screen...and this is the important bit ... You abort the password screen by using Android's back button (usually the left hand button at the bottom of the screen). DO NOT use the less then symbol '<' at the top left of the screen. So you have now backed out of the password screen and you are back at the list. Now click any protected note. You will find standard notes is unresponsive, you are not presented with the password screen or the note, nothing is displayed except the list of notes. The protected note will not open. Any unprotected notes open ok.

Now the only way to open a protected note is to kill standard notes by using the Android right hand button to swipe away the standard notes task. You can now open standard notes again and now you can click on a protected note and the password screen will now appear.

Interestingly, if instead of using Android's back button to abort the password entry screen you use the less than symbol '<' located to left of standard notes then the app doesn't become unresponsive and behaves correctly.

Hopefully you can reproduce this from my description above. If not I can always make a video that shows the problem.

Basically the standard Android back button (located bottom left of the screen) should behave in the same way as the '<' located at the top of the screen.

**Comment by radko93** [comment]
Hi, we recently released a new update to the app (3.5.1). Can you check if the issue also happens there?

**Comment by PartialVolume** [comment]
> Hi, we recently released a new update to the app (3.5.1). Can you check if the issue also happens there?

Hi, just updated to V3.5.1 (3000240) and can confirm that the issue no longer exists. Thanks 👍
